### ISSUE TITLE
Error text improvements

### ISSUE THREAD

**Issue by brendanarnold** [DFE-Digital/polis-whitelabel#17]
From the October Accessibility report ...

> You need to write in ‘Plain English’, don’t use uncommon words if possible. So your input warning of “Statement length exceeded by 1 characters” isn’t suitable, try changing it to “Too long, please make shorter by 1 character”. 

> Also any warnings need to come with advice on how to correct the issue, so the warning should be “Statement too long, please shorten before submitting”.

This relates to error messages in the statement submission, login and to the admin page inputs

**Comment by patcon** [comment]
I find this tool to be amaaaazing for supporting ppl to write accessible text: (top 1000 english words)
https://splasho.com/upgoer5/

It's a little silly, but it's a neat constraint to have the individually add words to the list, and perhaps kinda negotiate the need for each. Storing the word lists in the repo itself might allow that conversation to occur in-flow of PR process :)

<img width="1015" alt="Screen Shot 2022-12-09 at 6 33 06 PM" src="https://user-images.githubusercontent.com/305339/206769279-70ac1650-25d0-4412-af8d-9e754e8ea316.png">

---

BONUS: This could also be a CI linter, using the word list and a prose linting tool:
https://splasho.com/upgoer5/phpspellcheck/dictionaries/1000.dicin
http://proselint.com/

https://github.com/amperser/proselint/issues/1334
